internal/testrunner/runners/system/tester.go

```diff
@@ -750,7 +752,7 @@ func (r *tester) getDocs(ctx context.Context, dataStream string) (*hits, error)
 		return &hits{}, nil
 	}
 	if resp.IsError() {
-		return nil, fmt.Errorf("failed to search docs for data stream %s: %s", dataStream, resp.String())
+		return nil, fmt.Errorf("failed to search docs for index or data stream %s: %s", dataStream, resp.String())
```

Comment:
Should we keep just data stream to avoid being too much verbose here? 
`index` refers when the process waits for the docs to appear in transforms.